core/tests/webdriverio_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -523,10 +521,10 @@ describe('Core exploration functionality', function() {
 
     // Color grey when there is no warning, red when there is a warning.
     await explorationEditorSettingsTab.expectWarningsColorToBe(
-      'rgba(115, 115, 115, 1)');
+      'rgba(115,115,115,1)');
```

Comment:
Why the spaces were removed? Ditto below.

Reply:
`(await explorationObjectiveWarning.getCSSProperty('color')).value` returns color value without spaces unlike protractor. So we need to remove expect color values without spaces only.